README.md

```diff
@@ -1 +1,54 @@
-# Hello-World
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Hello Microverse Project
+
+> Description the project.
+> A Hello Microverse
+
+## Built With
+
+- Major languages HTML, CSS
+
+[Live Demo Link](https://github.com/Blessing600/Hello-Microverse)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Web Browser
+Code Editor
+
+### Setup
+
+```bash
+git clone https://github.com/Blessing600/Hello-Microverse
+cd Hello-Microverse
+```
+
+👤 **Author**
+
+- GitHub: [@Blessing600](https://github.com/Blessing600)
+- Twitter: [@Juliet1Ogbodo](https://twitter.com/Juliet1Ogbodo)
+- LinkedIn: [juliet-ezinne-a810921b5](https://www.linkedin.com/in/juliet-ezinne-a810921b5/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Great Job Creating Readme.MD file, but the template has been updated and you need to use the [new one](https://github.com/microverseinc/readme-template), kindly update it and make sure to fill in the necessary fields 🙏